modify.js

```diff
@@ -0,0 +1,17 @@
+var carLot = (function (carStuff) {
+  //removes highlight from previously clicked card
+  function removeHighlight() {
+    var removeHighlight = document.getElementsByClassName("carCard");
+    for (var i = 0; i < removeHighlight.length; i ++) {
+      removeHighlight[i].classList.remove('borderThick');
+    }
+  };
+
+    // adds background color and border to clicked card
+  function addBackgroundAndBorder (domEL, color) {
```

Comment:
One of the requirements was to pass in the domEl and the color. I would've expected you to eschew using the color, but the domEl at least you could pass in and use instead of relying on the 'event.target' within a function that it isn't declared.